src/css/betterttv-dark.css

```diff
@@ -944,8 +944,8 @@ div#site_header {
     color: #fff !important;
 }
 
-.ember-chat .chat-line .mentioning,
-.ember-chat .chat-line .mentioned {
+.chat-container .ember-chat .chat-line .mentioning,
+.chat-container .ember-chat .chat-line .mentioned {
```

Comment:
Careful.  `/chat` doesn't contain a `chat-container` class. Hence why I removed them [here](https://github.com/night/BetterTTV/pull/2270/commits/db93226969ab6298bb1f8f9d609daf24ad53f192).

Same for [L2184](https://github.com/EhsanKia/BetterTTV/blob/1e9b7c27348421fd9dcb8067090f468a55781f4b/src/css/betterttv.css#L2184) on betterttv.css